### DISCUSSION TITLE
Unable animate non-portaled Dialog

### DISCUSSION THREAD

**Discussion by SarKurd** [ariakit/ariakit#3912] [Questions] [answered]
I'm using the Dialog for a drawer and works greatly but in one of the use cases which i disable the portal feature to contain it absolutely within the relative parent, unable animating the slide in from right to left, but left to right works

Codesandbox: https://codesandbox.io/p/devbox/panel-slide-in-bug-6375lr

**Comment by diegohaz** [comment, marked as answer]
In your parent element, where you set `position: relative`, use `overflow: clip` instead of `overflow: hidden`.

**Reply by SarKurd** [reply]
Works perfectly, thanks.
Also to support older browser, you can use `clip-path: inset(0 0 0 0)`
